tests/test_notebooks.py

```diff
@@ -43,6 +45,22 @@ def test_eln_import(selenium_driver):
     driver.find_element(By.ID, "tooltip")
 
 
-def test_computational_resources(selenium_driver):
-    driver = selenium_driver("notebooks/computational_resources.ipynb")
+def test_computational_resources(notebook_service, selenium):
+    """Test the quicksetup of the code"""
+    url, token, docker_compose = notebook_service
+
+    # install the aiida-quantumespresso, we need it for test of some widgets
```

Comment:
Is this needed? Couldn't we just use one of the sample CalcJobs from `aiida-core`?

Reply:
Since this computational resource widget will load the plugin from aiida-code-registry, there is no "dummy" calcjobs from aiida-core. You can search for `default_calc_job_plugin` in https://aiidateam.github.io/aiida-code-registry/database_v2_1.json

Reply:
I don't think this is true. You should definitely be able to setup a code that is not in the registry. This is what I am doing with `aiida-orca`, which is not in the registry but the widget should still allow me to create it.

Reply:
Interesting, I make a test and it truly works without the entrypoint is installed. 
I think we can not install this, that's great!